chameleon/models.py

```diff
@@ -1,17 +1,56 @@
 from django.db import models
 from django.conf import settings
+from django.db.models import Q
+from django.core.exceptions import ObjectDoesNotExist
 
 class UserProperties(models.Model):
     user = models.OneToOneField(settings.AUTH_USER_MODEL, primary_key=True)
-    PI_STATUS_CHOICES = (
-        ('INELIGIBLE', 'Ineligible'),
+
+    class Meta:
+        verbose_name = ("User Properties")
+        verbose_name_plural = ("User Properties")
+
+    def pi_eligibility():
```

Comment:
Which of these two functions is being used? This one, or the one with the decorator? I like this one more because it's less implicit, but maybe it's unused?

Reply:
Yes, this one is unused, it's an alternative to using the decorator's more implicit design. I can update to use this method instead, should be a quick update and test.